### PR TITLE
GPU Support Table & Contributing Guide Fixes

### PR DESCRIPTION
**Pull request by microhobby**
In this series:

569604dacdadafa604ade0fb55e506c995ff6622: Fix some old links that are break, also update the contribute guid to open the MR against the `main` to `dev` branch;

565eac1f6bfe0c236c525e919f8f918f682828a3: Add the first version of the GPU support table for each of the GUI templates;

### CODE REVIEW

**Comment by microhobby** [comment]
Related-to: #51

File: GPU.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# GUI with HW Acceleration
+
+This document list the templates and the actual supported hardware acceleration.
+
+> ⚠️ These tables are WIP, feel free to send an MR if you have tested it on a machine that is not listed here.
+
+> ⚠️ Tested on the TorizonCore v6.2.0 with Bookworkm based images.
```
Comment (tronical) [inline GPU.md]:
```suggestion
> ⚠️ Tested on the TorizonCore v6.2.0 with Bookworm based images.
```

**Review by tronical — COMMENTED** [review]
Great idea to introduce this. I’m curious: what does OpenGL mean in this table (as opposed to OpenGL ES)? Are there Toradex devices where GL is preferred over GLES?

(I will update the Slint templates soon to add Vulkan support :)

**Comment by microhobby** [comment]
> Great idea to introduce this. I’m curious: what does OpenGL mean in this table (as opposed to OpenGL ES)? Are there Toradex devices where GL is preferred over GLES?
> 
> (I will update the Slint templates soon to add Vulkan support :)

Hey @tronical, about the OpenGL, no, there are no actual Toradex hardware products that support the Open GL "Desktop" version. But I think that it is important to list it there to make sure that the board listed support only OpenGL ES. Also, this will be useful for when we will list the third party `x86-64` boards running Torizon.

File: GPU.md (hunk 1 above)
Comment (microhobby) [reply GPU.md]: Thanks for the review!

**Comment by andreriesco** [comment]
LGTM

**Pull request merged**